### PR TITLE
Add CSS 2D/3D transform features

### PR DESCRIPTION
**Pull request by foolip**
Because transforms exist in multiple other parts of the platform
(DOMMatrix, canvas, SVG) prefix the names with "CSS".

### CODE REVIEW

File: feature-group-definitions/transforms2d.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+name: CSS 2D transforms
```
Comment (ddbeck) [inline feature-group-definitions/transforms2d.yml:1]: Given our recent changes to [the guidelines](https://github.com/web-platform-dx/web-features/tree/main/docs#feature-guidelines), what do you think of this approach instead?

```suggestion
name: 2D transforms (CSS)
```

File: feature-group-definitions/transforms3d.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+name: CSS 3D transforms
```
Comment (ddbeck) [inline feature-group-definitions/transforms3d.yml:1]:
```suggestion
name: 3D transforms (CSS)
```

File: feature-group-definitions/transforms2d.yml (hunk 1 above)
Comment (foolip) [reply feature-group-definitions/transforms2d.yml:1]: Maybe we just put CSS in the description?

Comment (ddbeck) [reply feature-group-definitions/transforms2d.yml:1]: Yes, that would be good too. I almost suggested it, but I figured that mean signing myself up to write them… which looked hard. 😅

**Comment by foolip** [comment]
@ddbeck I've added some descriptions, and it was hard. They could probably be improved.

Also I know we'll have some trouble when trying to compute a baseline status for 3D transforms, because parts of it like `backface-visibility` are quite new. Let's figure out splitting when we get to that.

**Review by ddbeck — APPROVED** [review]
I think these descriptions are pretty good! Let's go with it.

**Pull request merged**